README.md

```diff
@@ -21,127 +47,131 @@ c2patool is a command line tool for working with C2PA [manifests](https://c2pa.o
 | `application/c2pa`| `c2pa`      |    X      |
 | `video/quicktime` | `mov`       |           |
 
-## Installation
+NOTE: Quicktime (`.mov`) format is not yet fully supported.
 
-If you have [Rust](https://www.rust-lang.org/tools/install) installed, you can install or update c2patool using:
+## Usage
 
-```shell
-cargo install c2patool
+The tool's command-line syntax is:
+
+```
+c2patool [OPTIONS] [path]
 ```
 
-## Usage
+Where `<path>` is the path to the asset to read, or a JSON configuration file.
+
+The following table describes the command-line options.
+
+| CLI&nbsp;option&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; | Short version | Argument | Description |
+|-----|----|----|----|
+| `--config` | `-c` | `<config>` | Specifies a manifest definition as a JSON string. See [Providing a manifest definition on the command line](#providing-a-manifest-definition-on-the-command-line). |
+| `--detailed` | `-d` | N/A | Display detailed C2PA-formatted manifest data. See [Displaying a detailed manifest report](#displaying-a-detailed-manifest-report). |
+| `--force` | `-f` | N/A | Force overwriting output file. See [Forced overwrite](#forced-overwrite). |
+| `--help` | `-h` | N/A | Display CLI help information. |
+| `--output` | `-o` | `<output_file>` | Specifies path to output file. See [Adding a manifest to an asset file](#adding-a-manifest-to-an-asset-file). |
+| `--manifest` | `-m` | `<manifest_file>` | Specifies a manifest file to add to an asset file. See [Adding a manifest to an asset file](#adding-a-manifest-to-an-asset-file).
+| `--parent` | `-p` | `<parent_file>` | Specifies path to parent file. See [Specifying a parent file](#specifying-a-parent-file). |
+| `--remote` | `-r` | `<manifest_url>` | Specify remote manifest available over HTTP. See [Generating a remote manifest](#generating-a-remote-manifest)|
+| `--sidecar` | `-s` | N/A | Put manifest in external "sidecar" file with `.c2pa` extension. See [Generating an external manifest](#generating-an-external-manifest). |
+| `--version` | `-V` | N/A | Display version information. |
 
 ### Displaying manifest data
 
-Invoking the tool with a path to an asset will print a report describing the manifests contained in the file in JSON format to stdout.
+To display the manifest associated with an asset file, provide the path to the file as the argument; for example:
 
 ```shell
 c2patool sample/C.jpg
 ```
 
-### Detailed manifest report
+The tool displays the manifest JSON to standard output (stdout).
+
+To display a detailed report describing the internal C2PA format of manifests contained in the asset, use the `-d` option; for example:
 
-The `-d` or `--detailed` option will print a detailed report describing the internal C2PA format of manifests contained in the file in JSON format to stdout.
 ```shell
-c2patool sample/C.jpg -d
+c2patool -d sample/image.jpg
```

Comment:
sample/image.jpg doesn't have a manifest, so it won't show anything, sample/C.jpg has a manifest.